src/pages/RequestInvite/index.tsx

```diff
@@ -58,103 +58,86 @@ interface OwnProps {
 }
 
 type Props = OwnProps & {
-  authFullName: string | null;
+  loggedInFullName: string | null;
   postOperation: typeof postOperation;
-  fetchMemberByMidIfNeeded: typeof fetchMemberByMidIfNeeded;
-  fetchMemberByUidIfNeeded: typeof fetchMemberByUidIfNeeded;
-  isAuthLoaded: boolean;
-  isAuthMemberDocLoaded: boolean;
-  isToMemberDocLoaded: boolean;
-  memberId: string;
-  toMemberDoc: MemberDoc;
-  notSignedIn: boolean;
-  authFirebaseUser: { uid: string };
+  requestingFromMember: Member | undefined;
+  loggedInFirebaseUser: { uid: string } | null;
+
+  isLoading: boolean;
+  isOwnInvitePage: boolean;
 };
 
 export class RequestInvite extends React.Component<Props, State> {
-
   constructor(props: Props) {
     super(props);
     this.state = {
       videoUrl: null,
-      fullName: props.authFullName
+      fullName: props.loggedInFullName
     };
   }
 
   public componentWillReceiveProps(nextProps: Props) {
-    if (nextProps.authFullName !== this.props.authFullName) {
-      this.setState({ fullName: nextProps.authFullName });
+    if (nextProps.loggedInFullName !== this.props.loggedInFullName) {
+      this.setState({ fullName: nextProps.loggedInFullName });
     }
-    this.fetchIfNeeded(nextProps);
   }
 
-  private readonly fetchIfNeeded = (props: Props) => {
-    // TODO this logic very similar to Profile.componentWillReceiveProps.
-    // Decomp/https://reactjs.org/docs/higher-order-components.html/https://github.com/acdlite/recompose
-    if (props.memberId) {
-      this.props.fetchMemberByMidIfNeeded(props.memberId);
-    }
-    if (props.authFirebaseUser) {
-      this.props.fetchMemberByUidIfNeeded(props.authFirebaseUser.uid);
-    }
-  }
-
-  private readonly setVideoURL = (
-    event: React.FormEvent<HTMLInputElement>
-  ) => {
+  private readonly setVideoURL = (event: React.FormEvent<HTMLInputElement>) => {
     this.setState({ videoUrl: event.currentTarget.value });
   };
 
-  private readonly setAuthFullName = (
+  private readonly setloggedInFullName = (
```

Comment:
`setLoggedInFullName` (camelCase)

Although you can just get the value from the form element in the `handleOnSubmit` method to get rid of the component state, by doing the following: 

```
// in the component
<input
  name="fullName" //...
/>

// in handleOnSubmit
const fullName = event.currentTarget.elements["fullName"].value
```

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements

Reply:
Will look into this in a later PR. For now, I think the fullName stuff is getting set in multiple places, and I'm feeling a little uncomfortable about converting to an uncontrolled component without understanding it better. Will leave a TODO for a later PR.

Reply:
actually, jk, looked into it a bit more, taking it out.

Reply:
And reversing that. TSLint complained, don't feel like dealing with it right now :P